### DISCUSSION TITLE
Is this safe?

### DISCUSSION THREAD

**Discussion by vesera** [FabulousCupcake/umamusume-db-translate#27] [Q&A] [answered]
As the title said, is this really safe? I will not get banned if I uses this right? I can't really read Japanese so I somewhat need this. Also I am a bit unclear as to how to use it. If possible, please make it more clear for the how to use them. Thanks for your efforts for making this!

**Comment by FabulousCupcake** [comment, marked as answer]
The tool breaks TOS and can get you banned. Use your own judgement.
